modules/microshift-data-cleaning-ovn-data.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-cleanup-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-data-cleaning-ovn-data_{context}"]
+= cleaning the OVN data
+
+Run the following command to cleanup OVN data and to reset OVN settings after you change the network configuration.This command performs the following cleanup actions:
+
+* Stop and disable {microshift-short} services
+* Delete all {microshift-short} pods
+* Delete OVN configuration
+
+.Procedure
+
+. Run the following command to cleanup OVN data.
++
+[source,terminal]
+----
+$ sudo microshift-cleanup-data --ovn
+----
++
+.Example output
+[source,terminal]
+----
+Stopping MicroShift services
+Removing MicroShift pods
+Killing conmon, pause and OVN processes
+Removing OVN configuration
+MicroShift service was stopped
+Cleanup succeeded
+----
+
+. Run the following command to restart the {microshift-short} service.
```

Comment:
Please, copy the full text from the doc.

Reply:
done. note is added